infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -401,6 +401,22 @@ function staging_special_case__k8s_staging_cluster_api_gcp() {
     ensure_staging_gcb_builder_service_account "cluster-api-gcp" "k8s-infra-prow-build-trusted"
 }
 
+# In order to build the release artifacts, the group needs to be
+# able to create and manage a keyring to encrypt a secret token
+# that will be accessed and decrypted by a cloud build job.
+function staging_special_case__k8s_staging_kustomize() {
+    readonly STAGING_PROJECT="k8s-staging-kustomize"
+
+    local sa_email_group="k8s-infra-staging-kustomize@kubernetes.io"
+    local principal_group="serviceAccount:${sa_email_group}"
+    ensure_project_role_binding "${STAGING_PROJECT}" "${principal_group}" "roles/cloudkms.admin"
+
+    local sa_email_cloudbuild="660796270509@cloudbuild.gserviceaccount.com"
```

Comment:
Is this who the builds run as?  I see the following from `gcloud projects get-iam-policy k8s-staging-kustomize`

```yaml
bindings
- members:
  - serviceAccount:660796270509@cloudbuild.gserviceaccount.com
  - serviceAccount:gcb-builder@k8s-infra-prow-build-trusted.iam.gserviceaccount.com
  role: roles/cloudbuild.builds.builder
# ...
- members:
  - serviceAccount:service-660796270509@gcp-sa-cloudbuild.iam.gserviceaccount.com
  role: roles/cloudbuild.serviceAgent
# ...
```

IIRC triggering a build as a user is not the same as guaranteeing who that build runs as.

Can probably find this out from logs of recent builds

Reply:
Answering my own question: yes it is.  And annoyingly I could only find this on console, `gcloud builds describe` doesn't seem to dump this info?